.gitignore

```diff
@@ -1,2 +1,2 @@
 /examples/*/result
```

Comment:
We should also add a comment at the top mentioning that this file is autogenerated. Maybe add to File attribute `preamble` defaulting to something like `# Autogenerated file, do not edit`, it should fit majority of formats that we want to support and leave an escape hatch for those who need to override it.

This could probably be done in a separate PR though.

Reply:
I've explicitly added that to the text snippet for now

Reply:
Should work for this file for now. Could you please update this file though? It seems it doesn't match the definition now.

Reply:
Oh indeed, thanks. Fixed